openmdao/docs/features/recording/reading_data.rst

```diff
@@ -7,12 +7,23 @@ Accessing Recorded Data
 A :ref:`CaseReader<iterating_case_data>` provides access to the data for a particular case via a
 :class:`Case<openmdao.recorders.case.Case>` object.
 
-Getting Variable Data from a Case
----------------------------------
+`Case` objects have a number of attributes and methods for accessing variables and their values.
 
-`Case` objects have a number of attributes and methods to for accessing variables and their values.
+Getting Variable Data from Case Recording of a Driver
+-----------------------------------------------------
 
-The following example shows how to use these methods to easily check the variables of interest 
+Here are the methods to use when retrieving data from the recording of a `Driver` case recorder.
```

Comment:
These methods should be valid for any type of recorder, I believe.  
VOIs are pulled from the appropriate system based on the `recording_requester`, per:
https://github.com/OpenMDAO/OpenMDAO/blob/master/openmdao/recorders/sqlite_recorder.py#L286